aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -106,11 +106,19 @@ pub fn generate_feature_upgrade_proposal(
                     writer,
                     "features::change_feature_flags(framework_signer, enabled_blob, disabled_blob);"
```

Comment:
n00b: where the signer gets generated?

Reply:
Signer is generated from resolving governance proposal.